Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -101,12 +78,19 @@ int RunFuzzTestingMain() {
   // The corpus location for the fuzzing target.
   NSString *corpus_location;
 
+  // Fuzzing target method, equivalent to LLVMFuzzerTestOneInput. This variable
+  // holds a pointer to the fuzzing method that is called by the fuzzing driver
+  // thousands of times with different inputs. Any method assigned to this
```

Comment:
I'd change "thousands of times" to "repeatedly".

Reply:
Yes, it sounds better, thanks.